User.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UserStructure.swift
+//  ModelModel
+//
+//  Created by Stanislav Cherkasov on 12.12.17.
+//  Copyright © 2017 Stanislav Cherkasov. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class User {
```

Comment:
I guess struct is more appropriate here, also you have default init for struct and can omit init with { self.name = name ... } constructions altogether